.github/workflows/continuous-integration.yml

```diff
@@ -42,14 +42,15 @@ jobs:
         dependencies:
           - "highest"
         extension:
+          - "sqlite3"
           - "pdo_sqlite"
         include:
           - os: "ubuntu-20.04"
             php-version: "8.1"
             dependencies: "lowest"
             extension: "pdo_sqlite"
           - os: "ubuntu-22.04"
-            php-version: "8.1"
+            php-version: "8.4"
```

Comment:
Both conflicts in this file emanate from https://github.com/doctrine/dbal/pull/6740

I'm not sure if this is the right conflict resolution or if I should stick with 8.1 here.

Reply:
I think 8.4 is the right choice as https://github.com/doctrine/dbal/pull/6740 made this change. IIRC, I resoved it the same when I merged it to 4.2.x.